components/properties/src/script.rs

```diff
@@ -45,18 +44,98 @@ impl ScriptWithExt {
 }
 
 impl ScriptWithExt {
```

Comment:
Issue: `ScriptWithExt` is not "really" a public-facing type.  Its only use in a public API is in the ScriptExtensions hidden constructor function.  I think you should mark `ScriptWithExt` as hidden, too.

Reply:
Done. I marked both impl blocks as hidden in docs. I also went ahead and marked the struct itself as hidden, too, because we're also marking its only public usage (the ScriptExtensions constructor) as hidden in this PR.